docs/samples/contact-center/app.js

```diff
@@ -189,6 +186,16 @@ function register() {
     }) 
 }
 
+function populateWrapupCodesDropdown() {
```

Comment:
We need name and id for the payload for wrapupm hence we store only that as of now.